app/frontend/src/features/messages/Messages.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import wrapper from "../../test/hookWrapper";
+import user from "../../test/fixtures/defaultUser.json";
+import { service } from "../../service";
+import { CouchRequestsNotification, MessagesNotification } from "./Messages";
+
+const pingMock = service.api.ping as jest.Mock<
+  ReturnType<typeof service.api.ping>,
+  []
+>;
+
+afterEach(() => {
+  jest.restoreAllMocks();
+});
+
+describe.each`
+  name                           | label            | count   | Component
+  ${"CouchRequestsNotification"} | ${"Hosting"}     | ${"99"} | ${CouchRequestsNotification}
+  ${"MessagesNotification"}      | ${"Group Chats"} | ${"99"} | ${MessagesNotification}
```

Comment:
:O

Reply:
I rarely came across a good use case for this test feature in jest until this 😄